src/Tools/Common/Commands/Utils.cs

```diff
@@ -53,7 +53,17 @@ public static int FindProcessIdWithName(string name)
         public static bool ValidateArguments(int processId, string name, string port, out int resolvedProcessId)
         {
             resolvedProcessId = -1;
-            if (processId != 0 && name != null && !string.IsNullOrEmpty(port))
+            if (processId == 0 && name == null)
+            {
+                Console.WriteLine("Must specify either --process-id, --name, or --diagnostic-port.");
+                return false;
+            }
+            else if (processId < 0)
+            {
+                Console.WriteLine($"{processId} is not a valid process ID");
+                return false;
+            }
+            else if (processId != 0 && name != null && !string.IsNullOrEmpty(port))
             {
                 Console.WriteLine("Only one of the --name, --process-id, or --diagnostic-port options may be specified.");
```

Comment:
We are only checking for all three set, but we are saying only one can be set.

Reply:
Yes, only one of these can be specified. This is checking for the case where all three are specified and returning false -- there are subsequent if checks below this that checks for 2 of them being specified and output the corresponding error message depending on which 2 are specified.